README.md

```diff
@@ -0,0 +1,161 @@
+Triage
+======
+
+Risk modeling and prediction for public policy
+
+[![image](https://travis-ci.org/dssg/triage.svg?branch=master)](https://travis-ci.org/dssg/triage)
+[![image](https://codecov.io/gh/dssg/triage/branch/master/graph/badge.svg)](https://codecov.io/gh/dssg/triage)
+[![image](https://codeclimate.com/github/dssg/triage.png)](https://codeclimate.com/github/dssg/triage)
+
+Predictive analytics projects require the coordination of many different
+tasks, such as feature generation, classifier training, evaluation, and
+list generation. These tasks are complicated in their own right, but in
+addition have to be combined in different ways throughout the course of
+the project.
+
+Triage aims to provide interfaces to these different phases of a
+project, such as an `Experiment`. Each phase is defined by configuration
+specific to the needs of the project, and an arrangement of core data
+science components that work together to produce the output of that
+phase.
+
+`Experiment` (create features and models) -> `Audition` (pick the best models) -> `Postmodeling` (dive into best models)
+
+## Documentation Quick Links
+
+- [Dirty Duck Tutorial](https://dssg.github.io/triage/dirtyduck/docs/)
+- [Triage Documentation Site](https://dssg.github.io/triage/)
+
+## Getting Started
+
+### Prerequisites
+
+To use Triage, you first need:
+
+- Python 3.6
+- A PostgreSQL database with your source data (events, geographical data, etc) loaded.
+- Ample space on an available disk, (or for example in Amazon Web Services's S3), to store the needed matrices and models for your experiments
+- A question you want to answer!
+
+### Install
+
+Triage is a Python package distributable via `setuptools`. It may be
+installed directly using `easy_install` or `pip` (`pip install triage`), or named as a
+dependency of another package as `triage`.
+
+### Design an Experiment
```

Comment:
Maybe briefly describe what an experiment is in terms of triage (and maybe why we call it that) since it's not necessarily familiar/standard terminology.

Reply:
Since we kind of do this later, perhaps it makes sense to move 'Triage Phase Details' up before this part? Or even just stick this section into the 'Experiment' part of the aforementioned 'Triage Phase Details'?